addons/forum-quote-code-beautifier/addon.json

```diff
@@ -3,10 +3,14 @@
   "description": "Shows nicer code and quote blocks on the forums.",
   "credits": [
     {
-      "name": "GrahamSH (Port)"
+      "name": "GrahamSH",
+      "note": "addon",
```

Comment:
"port" sounds better imo

Reply:
port, where ships load? tcp ports?

Reply:
hmm i see your point. maybe `port to addon`?

Reply:
"Addon version"?

Reply:
I'm going with "addon version"